modules/util/portal-tools-rest-builder-test-impl/rest-openapi.yaml

```diff
@@ -880,6 +880,41 @@ paths:
                     description:
                         "OK"
             tags: ["TestEntity"]
+    /test-entities/multiform/bulk:
+        post:
+            operationId: postTestEntityMultiformBulk
+            requestBody:
+                content:
+                    multipart/form-data:
+                        schema:
+                            properties:
+                                testEntities:
+                                    items:
+                                        $ref: "#/components/schemas/TestEntity"
+                                    type: array
+                            type: object
+            responses:
+                200:
+                    content:
+                        "application/json":
+                            schema:
+                                properties:
```

Comment:
This is not adding any value to the test. In order to simplify it, we could:

- Remove this schema response and just return 200
- Return the List of TestEntity that have been created

I vote for the first one